### ISSUE TITLE
Remove .git and other ignored files from largest files table

### ISSUE THREAD

**Issue by soundasleep** [soundasleep/statgit2#7]
It's accidentally capturing files in .git:

```
.git/objects/pack/pack-c981d6fdg...2c7e1.pack	1	0	306,514.41 KB
.git/objects/pack/pack-c981d6fdg...2c7e1..idx	1	0	20,681.78 KB
```

**Comment by soundasleep** [comment]
May also need to delete all existing Files which start with .git, because there are some in the database

**Comment by soundasleep** [comment]
Resolved in PR#14
